src/workflows-app/components/WorkflowsAppNavPanel.ts

```diff
@@ -338,7 +339,6 @@ export const WorkflowsAppNavPanel = ({
               h(ImportGithub, {
                 setLoading,
                 signal,
-                onDismiss: null,
```

Comment:
there is no exposed onDismiss prop on ImportGithub component... I'm guessing this was a remnant of an earlier code state, or just something that lack of type safety in .js let sneak in.